code/core/src/mocking-utils/automock.ts

```diff
@@ -8,11 +8,12 @@ import type {
 } from 'estree';
 import MagicString from 'magic-string';
 
-import { __STORYBOOK_GLOBAL_THIS_ACCESSOR__ } from '../presets/vitePlugins/vite-inject-mocker/constants';
 import { type Positioned, getArbitraryModuleIdentifier } from './esmWalker';
 
 type ParseFn = (code: string) => Program;
 
+export const __STORYBOOK_GLOBAL_THIS_ACCESSOR__ = '__vitest_mocker__';
```

Comment:
This is a vitest reference in common utils. Is that intentional or perhaps unavoidable?